### PR TITLE
text.last-index-of

### PR DESCRIPTION
**Pull request by EugeneDar**
https://github.com/objectionary/eo-strings/issues/66

### CODE REVIEW

**Comment by EugeneDar** [comment]
@Graur @levBagryansky please have a look

**Review by Graur — APPROVED** [review]
@EugeneDar Looks good to me

**Comment by Graur** [comment]
@yegor256 Please, take a look

**Comment by levBagryansky** [comment]
@EugeneDar sure

File: src/main/eo/org/eolang/txt/text.eo [hunk 1]
```diff
@@ -125,7 +128,23 @@
   # @todo #43:30min Current implementation using Java.
```
[the remark this answers is not in the document]
Comment (EugeneDar) [reply src/main/eo/org/eolang/txt/text.eo:128]: @yegor256 yes, I'll fix it.

**Comment by EugeneDar** [comment]
@yegor256 please have a look again

**Pull request merged**